zzhoritontalprogressbar/src/main/java/me/zhouzhuo/zzhoritontalprogressbar/ZzHorizontalProgressBar.java

```diff
@@ -0,0 +1,1182 @@
+package me.zhouzhuo.zzhoritontalprogressbar;
+
+import ohos.agp.components.AttrSet;
+import ohos.agp.components.Component;
+import ohos.agp.components.Component.DrawTask;
+import ohos.agp.render.Canvas;
+import ohos.agp.render.LinearShader;
+import ohos.agp.render.Paint;
+import ohos.agp.render.Shader.TileMode;
+import ohos.agp.utils.Color;
+import ohos.agp.utils.Point;
+import ohos.agp.utils.RectFloat;
+import ohos.app.Context;
+import ohos.hiviewdfx.HiLog;
+import ohos.hiviewdfx.HiLogLabel;
+import com.hmos.compat.utils.AttrUtils;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/**
+ * 水平进度条*
+ * Created by 周卓 on 2016/9/22.
+ */
+
+public class ZzHorizontalProgressBar extends Component implements DrawTask {
+
+
+    private final HiLogLabel hiLogLabel = new HiLogLabel(0, 0, "ZzHorizontalProgressBar");
```

Comment:
make this a static final variable and use the approprate naming convention.

Reply:
done